uni/ulsp/symbols.icn

```diff
@@ -0,0 +1,202 @@
+               class_table["range"]["start"][2],
+               class_table["range"]["end"][1],
+               class_table["range"]["end"][2],
+               class_table["selectionRange"]["start"][1],
+               class_table["selectionRange"]["start"][2],
+               class_table["selectionRange"]["end"][1],
+               class_table["selectionRange"]["end"][2],
+               []
+            )
+         }
+
+         # Get methods inside classes from filehandler
+         every method_name := key(filehandler.getInternalClasses()[class_name]["methods"]) do {
+            if member(pos_table["methods"], method_name) then {
+               method_table := pos_table["methods"][method_name]
+
+               method_symbol := constructResult(
+                  method_name,
+                  6,
+                  method_table["range"]["start"][1],
+                  method_table["range"]["start"][2],
+                  method_table["range"]["end"][1],
+                  method_table["range"]["end"][2],
+                  method_table["selectionRange"]["start"][1],
+                  method_table["selectionRange"]["start"][2],
+                  method_table["selectionRange"]["end"][1],
+                  method_table["selectionRange"]["end"][2],
+                  []
+               )
+               entity_table[method_name] := method_symbol
+               put(class_symbol["children"], method_symbol)
+            }
+         }
+
+         # Get initially sections inside classes from filehandler
+         if member(pos_table, "initially") && member(pos_table["initially"], class_name) then {
+            initially_table := pos_table["initially"][class_name]
+
+            method_symbol := constructResult(
+               "initially",
+               9,
+               initially_table["range"]["start"][1],
+               initially_table["range"]["start"][2],
+               initially_table["range"]["end"][1],
+               initially_table["range"]["end"][2],
+               initially_table["selectionRange"]["start"][1],
+               initially_table["selectionRange"]["start"][2],
+               initially_table["selectionRange"]["end"][1],
+               initially_table["selectionRange"]["end"][2],
+               []
+            )
+            entity_table["initially" || class_name] := method_symbol
+            put(class_symbol["children"], method_symbol)
+         }
+
+         put(results_list, class_symbol)
+      }
+
+      # Get variables from filehandler
+      every variable_name := key(filehandler.getVariables()) do {
+         if member(pos_table["variables"], variable_name) then {
+            variable_table := pos_table["variables"][variable_name]
+            sl := variable_table["selectionRange"]["start"][1]
+            sc := variable_table["selectionRange"]["start"][2]
+            el := variable_table["selectionRange"]["end"][1]
+            ec := variable_table["selectionRange"]["end"][2]
```

Comment:
In this case and all similar code blocks above and below, you gotta decide if it is cheaper/faster to store the first table  lookup in a local variable instead of doing it multiple times.  
```
selection_range := variable_table["selectionRange"] 
sl := selection_range ["start"][1]
...
```
And, at least in this code block , I don't see any use for `variable_table` outside these lookups, so you may want to just skip `variable_table` altogether and go directly to the item you care about, so the above code becomes : 
```
selection_range := pos_table["variables"][variable_name]["selectionRange"] 
sl := selection_range ["start"][1]
...
```